browser/ui/views/toolbar/brave_toolbar_view_browsertest.cc

```diff
@@ -78,8 +84,15 @@ class BraveToolbarViewTest : public InProcessBrowserTest {
     return button->GetVisible();
   }
 
+  bool is_bookmark_button_shown() {
+    BookmarkButton* bookmark_button = toolbar_view_->bookmark_button();
+    DCHECK(bookmark_button);
```

Comment:
seems weird to have a `DCHECK` here instead of an `ASSERT_NE(bookmark_button, nullptr)`, but I suppose tests should always be run with DCHECK on and same thing is done higher up so 🤷